material-overrides/assets/stylesheets/polkadot.css

```
@@ -1513,34 +1513,74 @@ div.mermaid {
   font-size: 12px;
 }
 
-/** Where to Go Next Badge styling */
-.md-typeset span.badge {
+/** Badge Styling */
+/* Basic style */
+span.badge {
   border-radius: 0.5em;
   font-size: 0.8em;
   padding: 0.2em 0.5em;
-  margin-right: 0.5em;
 }
 
-.md-typeset span.badge.tutorial {
+/* Color Variants */
+span.badge.tutorial {
   background-color: var(--electric-violet);
   color: var(--white);
 }
-
-.md-typeset span.badge.guide {
+span.badge.guide {
```

There should be line breaks between each block of styles

```suggestion

span.badge.guide {
```